_query-dsl/specialized/rank-feature.md

```diff
@@ -0,0 +1,547 @@
+---
+layout: default
+title: Rank feature
+parent: Specialized queries
+nav_order: 75
+---
+
+# Rank feature
+
+Use the `rank_feature` query to boost document scores based on numeric values in the document, such as relevance scores, popularity, or freshness. This query is ideal if you want to fine-tune relevance ranking using numerical features. Unlike [full-text queries]({{site.url}}{{site.baseurl}}/query-dsl/full-text/index/), `rank_feature` focuses solely on a numeric signal, and is most effective when combined with other queries in a compound query like `bool`.
+
+The `rank_feature` query expects the target field to be mapped as a [`rank_feature` field type]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/rank/). This enables internally optimized scoring for fast and efficient boosting.
+
+The score impact depends on the field value and the optional `saturation`, `log` or `sigmoid` function used.
+
+## Parameters
+
+| Parameter               | Required/Optional | Description                                                                                                              |
+| ----------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `field`                 | Required          | A `rank_feature` or `rank_features` field that contributes to document scoring.|
+| `boost`                 | Optional          | A multiplier applied to the score. Default is `1.0`. Values between 0 and 1 reduce the score, values above 1 amplify it. |
+| `saturation`            | Optional          | Applies a saturation function to the feature value. Boost grows with value but levels off beyond the `pivot`. (Default function if no other function is provided)|
+| `log`                   | Optional          | Uses a logarithmic scoring function based on the field value. Best for large ranges of values.|
+| `sigmoid`               | Optional          | Applies a sigmoid (S-shaped) curve to score impact, controlled by `pivot` and `exponent`.|
+| `positive_score_impact` | Optional          | When `false`, lower values score higher. Useful for features like price where smaller is better. Defined as part of the mapping. (Default is `true`)|
```

Comment:
I believe that these functions are applied to field values at query time. If that is the case can we explicitly state these facts ?